lib/DDG/Spice/Arxiv.pm

```diff
@@ -0,0 +1,21 @@
+package DDG::Spice::Arxiv;
+
+# ABSTRACT: Retrieve articles from arxiv.org by identifier
+
+use DDG::Spice;
+
+spice is_cached => 1;
+spice proxy_cache_valid => "200 7d";
+
+spice wrap_jsonp_callback => 1;
+
+spice to => 'https://duckduckgo.com/x.js?u=http://export.arxiv.org/api/query?id_list=$1';
+
+triggers query_lc => qr/^arxiv[:\s]+?([^\s]+)/;
```

Comment:
@zachthompson fwiw this is one of the rare cases where using a word trigger, and splitting words on `-`, `:`, etc. is actually useful. Maybe we should have flag to force matching words on delimiters so the default case is to match on just `\s`?

@jbarrett did you try a word trigger with `arxiv`? I have a feeling it'll work here and then we can do something like:

```perl
handle query_lc => sub {
    return unless m/^arxiv[:\s]+?([^\s]+)/;
    return $1;
}
```

I may be wrong though...